src/main/java/games/strategy/triplea/ui/MovePanel.java

```diff
@@ -1346,6 +1346,7 @@ protected final void cleanUpSpecific() {
     selectedUnits.clear();
     updateRouteAndMouseShadowUnits(null);
     forced = null;
+    getMap().showMouseCursor();
```

Comment:
I'm guessing this is a slightly unrelated fix, that at some point prior this call got dropped?